codacy/values.yaml

```diff
@@ -70,6 +70,8 @@ global:
         name: "coverage"
       cliReports:
         name: "cli-reports"
+      commitDiff:
+        name: "commit-diff"
```

Comment:
Shouldn't we use `https://github.com/helm/charts/tree/master/stable/minio#configuration` to create the buckets instead of having special code to do it?

Reply:
For compatibility reasons with legacy-enterprise we still need to create the buckets :/

Reply:
not really, if it is running it should exist by now.

Reply:
What do you mean @rtfpessoa ? 
In legacy-enterprise we are creating the buckets with the component, right?

Reply:
yup, but we could do this same improvement there. This is just a variable, isn't this supported by the docker itself?

Still even if it is not doing it now should make it easier for the future.

Reply:
ok.. so the suggestion is to create all the buckets when minio is created, by putting the ".values.minio.buckets = xxxx" in the values file

<img width="843" alt="image" src="https://user-images.githubusercontent.com/3610528/73734051-ef7c6580-4734-11ea-8015-ed4f152192f9.png">

If the bucket is created, the component is going to detect that and just use that bucket instead of creating one.

Is that it?

Reply:
yes

Reply:
@rtfpessoa, @joaocosta-codacy is implementing your request in another PR, on top of this one. I'll resolve this comment.